docs/devdocs/native.md

```diff
@@ -20,7 +20,21 @@ limitations under the License.
 # Initial testing of native 
 
 ## Compile on linux
-JAVA_INCLUDE=/usr/lib/jvm/java-1.8.0-openjdk/
-g++ -I/usr/local/include/eigen3/ -I$JAVA_INCLUDE/include/ -I$JAVA_INCLUDE/include/linux/ -shared -Wl,--no-undefined -fPIC -fopenmp -msse2 -O2  -o libSystemML.so  SystemML.cpp
+Uses CMake.
+
+Set JAVA\_HOME to your java installation. For instance, on Ubuntu, with Oracle Java 8, JAVA\_HOME=/usr/lib/jvm/java-8-oracle
+Set EIGEN3\_INCLUDE\_DIR to where the Eigen3 header files can be found. For instance, on my machine, EIGEN3\_HOME\_DIR=/home/njindal/git/eigen
```

Comment:
Can u change /home/njindal/git/eigen to /usr/local/include/eigen3/ or something like that, else we might forget it while creating PR ?